qase-testng/src/test/java/io/qase/testng/QaseListenerTest.java

```diff
@@ -57,6 +57,23 @@ public void passedBulkTest() {
                         "}")));
     }
 
+    @Test
+    public void passedWithStepsBulkTest() {
+        useBulk(true);
+        runTest(WithSteps.class);
+        verify(postRequestedFor(urlPathEqualTo("/v1/result/PRJ/777/bulk"))
+                .withHeader("Token", equalTo("secret-token"))
+                .withHeader("Content-Type", equalTo("application/json; charset=UTF-8"))
+                .withRequestBody(equalToJson("{\n" +
```

Comment:
I would suggest writing Json into a separate object and then comparing it with an object, not a huge json

Reply:
I don't know how to do it in wiremock, only if I will marshal object through gson or jackson and after it deserialize object to json. But I think it isn't a good idea.
Can you text how you see it?

Reply:
I was moving to another city, I could not answer earlier, sorry.
Previously, it seems, I did not use WireMock, now I looked at the docks on it and also did not immediately find how to do it normally. I assumed that you can "shove" json into an object and compare through the equals method with it.
Apparently not.